src/backend/app/projects/project_routes.py

```diff
@@ -1180,27 +1180,27 @@ async def get_template_file(
     "/project_dashboard/{project_id}", response_model=project_schemas.ProjectDashboard
 )
 async def project_dashboard(
-    project_id: int,
     background_tasks: BackgroundTasks,
+    project: project_deps.get_project_by_id = Depends(),
```

Comment:
The my other comment.

The class after the `:` is for type hinting, so is specifying what type of value `project` is.

The Depends is used after `=`, with the dependency inside the brackets.